driver-core/src/main/com/mongodb/internal/connection/DefaultServer.java

```diff
@@ -201,7 +205,8 @@ public void close() {
             connectionPool.close();
             serverMonitor.close();
             isClosed = true;
-            serverListener.serverClosed(new ServerClosedEvent(serverId));
```

Comment:
Just curious why you decided to break this up when it doesn't seem related to the rest of your pr?

Reply:
Good catch.  It's an artifact of a previous design that I later rejected.  I will revert.

Reply:
Done. Please resolve if you like the change